src/gui/static/src/app/components/pages/send-skycoin/send-skycoin.component.spec.ts

```diff
@@ -1,25 +1,140 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import * as sinon from 'sinon';
+import { async, inject, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { HttpModule, XHRBackend } from '@angular/http';
+import { MdCardModule, MdSelectModule, MdSnackBarModule, MdTooltipModule } from '@angular/material';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { RouterTestingModule } from '@angular/router/testing';
+import { MockBackend } from '@angular/http/testing';
 
+import { NgxDatatableModule } from '@swimlane/ngx-datatable';
+import { ButtonComponent } from '../../layout/button/button.component';
 import { SendSkycoinComponent } from './send-skycoin.component';
+import { DateTimePipe } from '../../../pipes/date-time.pipe';
+import { SkyPipe } from '../../../pipes/sky.pipe';
+import { ApiService } from '../../../services/api.service';
+import { WalletService } from '../../../services/wallet.service';
 
 describe('SendSkycoinComponent', () => {
   let component: SendSkycoinComponent;
+  let mockBackend: MockBackend;
   let fixture: ComponentFixture<SendSkycoinComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ SendSkycoinComponent ]
+      declarations: [
+        ButtonComponent,
+        DateTimePipe,
+        SendSkycoinComponent,
+        SkyPipe
+      ],
+      imports: [
+        HttpModule,
+        MdCardModule,
+        MdSelectModule,
+        MdSnackBarModule,
+        MdTooltipModule,
+        NgxDatatableModule,
+        NoopAnimationsModule,
+        ReactiveFormsModule,
+        RouterTestingModule
+      ],
+      providers: [
+        ApiService,
+        WalletService,
+        { provide: XHRBackend, useClass: MockBackend }
```

Comment:
__note:__ the only mock/stub we need here is actually HTTP, the rest can be tested as is.